database/seed.js

```diff
@@ -39,11 +51,27 @@ let seedMainProduct = function () {
 let seedPhotos = function() {
     var promisearr = [];
 
+    // FUNCTION TO MATCH PHOTO ID TO PRODUCT ID
+    let matchID = function(number) {
+    if (number === 0) {
+        return 0
+    }
+    let dividedNumber = number / 10;
+    let intendedNumber = Math.floor(dividedNumber);
+    return intendedNumber;
+}
+
     for (var j = 0; j < 100; j++) {
-        let randomPhotoUrl = faker.image.imageUrl(); // https://www.aws.photo.com
+        // let randomPhotoUrl = faker.image.imageUrl(); // https://www.aws.photo.com
+
+        let randomPhotoUrl = `https://loremflickr.com/500/400/products?random=${j}`
+
+        let productID = matchID(j)
 
         let photostable = new photos({
-            photo_url: `${randomPhotoUrl}`
```

Comment:
this seed should be refactored to use the amazon urls instead of these old loremflickr ones